src/index.html

```diff
@@ -10,23 +10,27 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Two Search Bars</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-container">
+      <input
+        type="text"
+        class="search-bar big"
+        placeholder="Search..."
+        data-qa="big"
+        data-qa-keypress="key-detected">
```

Comment:
The `data-qa-keypress` attribute now has a value `key-detected`. Please verify if this is the intended functionality or if it needs further adjustment.